proto/controlclient.proto

```diff
@@ -55,3 +56,17 @@ message _RevokeSigningKeyRequest {
 
 message _RevokeSigningKeyResponse {
 }
+
+message _SigningKey {
+  string key_id = 1;
+  uint64 expires_at = 2;
+}
+
+message _ListSigningKeysRequest {
+  string next_token = 1; 
```

Comment:
I remain saddened that we don't use streaming responses for these.

Reply:
> I remain saddened that we don't use streaming responses for these.

Yeah, I would like that too and it seems like we aren't using pagination atm. I wanted to keep this here to at least maintain similar patterns